cheatsheets/ssrf.md

```diff
@@ -14,6 +18,13 @@ https://520968996
 
 _Note:_ The latter can be calculated using http://www.subnetmask.info/
 
+
+[**IP-Obfuscator**](https://github.com/C-REMO/Obscure-IP-Obfuscator)
```

Comment:
Do not use a Markdown link for this. Just place the link (https://github.com/C-REMO/Obscure-IP-Obfuscator) directly into the document.